Examples/Io/EDM4hep/src/EDM4hepUtil.cpp

```diff
@@ -277,10 +278,10 @@ void EDM4hepUtil::writeTrajectory(
 
     edm4hep::TrackState trackState;
 
-    // This makes the hard assumption that |q| = 1
-    Acts::SingleBoundTrackParameters<Acts::SinglyCharged> parObj{
-        state.referenceSurface().getSharedPtr(), state.parameters(),
-        state.covariance()};
+    // TODO get particle hy
```

Comment:
```suggestion
    // TODO get particle hypothesis
```
We'd have to store this somehow, which I'm not sure is easily possible.